components/commander/common/prefs.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_COMMANDER_COMMON_PREFS_H_
+#define BRAVE_COMPONENTS_COMMANDER_COMMON_PREFS_H_
+
+#include "base/component_export.h"
+
+namespace commander::prefs {
+
+COMPONENT_EXPORT(COMMANDER_COMMON)
+extern const char kCommanderFrecencies[];
```

Comment:
`constexpr char kCommanderFrecencies[] = "brave.commander.command_frecencies";` w/o cc file.
also recommend to use `pref_names.h` as a file name. maybe it's our(chromium) name convention  for this kind of file.

Reply:
I think `constexpr` doesn't play nice with `COMPONENT_EXPORT`

Reply:
Ok, I didn't know that :)